packages/connect-examples/expo-example/src/components/ui/AutoWrapperTextArea.tsx

```
@@ -1,21 +1,32 @@
-    const scrollHeight = get(textAreaRef, 'current.scrollHeight', 0);
+  // 使用useCallback优化调整高度的函数
+  const adjustHeight = useCallback(() => {
+    if (!textAreaRef.current) return;
+
+    const scrollHeight = textAreaRef.current.scrollHeight || 0;
     if (scrollHeight > 0) {
-      // @ts-expect-error
-      textAreaRef.current.style.height = 'inherit';
-      // @ts-expect-error
-      textAreaRef.current.style.height = `${textAreaRef.current.scrollHeight}px`;
+      const currentHeight = parseInt(textAreaRef.current.style.height);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)